lib/services/api/user_token_service.rb

```diff
@@ -23,17 +23,19 @@ def api_config
       @api_config ||= ::Settings[base_config[:module]].to_hash
     end
 
-    def generate_token(userid, requester_type)
+    def generate_token(userid, requester_type, token_ttl = nil)
```

Comment:
Use a keyword argument to make callers more explicit and allow for easier additions in the future?

Reply:
what would we do for backward compatibility if we use keyword argument, we would have to change all callers to use the keyword argument style

Reply:
huh? Can you tell me a hypothetical example of what you mean? I'm talking about using a keyword arg for `token_ttl` specifically, not the other two, if that isn't clear. See example in my comment below in the tests. The token_ttl is a defaulted, optional parameter (as opposed to the other two) which usually presents itself well as a keyword argument.

Reply:
Usage would be same as today:

```
UserTokenService.new.generate_token(userid, 'api')
```

With the keyword arg, if you need to pass an optional token, you'd pass by name:

```
UserTokenService.new.generate_token(userid, 'api', token_ttl:  ansible_playbook_ttl)
```

Reply:
Yup! ❤️